pom.xml

```diff
@@ -54,6 +73,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-rest-client-jsonb</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.camel.quarkus</groupId>
+      <artifactId>camel-quarkus-gson</artifactId>
+    </dependency>
```

Comment:
When you use mvn dependency:tree you will see jackson is referred indirectly,which means it will also be included in the final uber jar. So I recommend to use it here and add it as direct mvn dep.